### ISSUE TITLE
Winget: chezmoi installed via winget in windows not added to the path

### ISSUE THREAD

**Issue by dhruvinsh** [twpayne/chezmoi#3039]
## Describe the bug
Chezmoi installation via winget, doesn't get added to the path
```
winget install twpayne.chezmoi
```

## Expected behavior

It should be added to the path automatically so that `chezmoi` executable works without needing to making manual changes in the path.

## Output of command with the `--verbose` flag

NA

**Comment by twpayne** [comment]
Isn't this a problem with Winget, rather than chezmoi? Surely Winget is clever enough to add the installed binary to the path, or update the path to include the directory where it installed the binary? Or am I mistaken?

chezmoi's Winget support is in https://github.com/twpayne/chezmoi/blob/7d11c382beb07278a1cd6fef5f23948ac875fa03/.github/workflows/winget.yml.

**Comment by dhruvinsh** [comment]
> Isn't this a problem with Winget, rather than chezmoi?

Yes you are right. After looking further, and trying other packages I realized that it would be more of winget issue, It does good job isolating the package in the separate directory but doesn't set the path automatically. Similar behaviour I observed for the syncthing. I think for me it would be better to use scoop (really good in my opinion) or manual installation.  

Some lengthy conversion is here: https://github.com/microsoft/winget-cli/issues/549
